static/slider.js

```diff
@@ -201,9 +265,44 @@
         elem[0].dataset.stars = size;
     }
 
+    /**
+     * This is needed when sublevels were merged vertically and horizontally
+     * and then unmerged horizontally.
+     * @param {Integer} size - Quantity of observations.
+     * @param {String} color - New color for sublevel.
+     * @param {String} starsClass - Class to find star element.
+     * @param {String} heatClass - Class to find heat element.
+     * @param {String} construct - ID of current construct.
+     * @param {Integer} sublevel - ID of sublevel.
+     */
+    function changeElemsUnmerge(size, color, starsClass, heatClass, construct, sublevel) {
```

Comment:
This method seperates level merged vertically and horizontally, right? `changeElemsUnmerge` isn't a good name.